mokapot/plugins/__init__.py

```diff
@@ -5,10 +5,7 @@
 
 import pandas as pd
 
-try:
-    from importlib.metadata import entry_points
-except ImportError:
-    from importlib_metadata import entry_points
+from importlib_metadata import entry_points
```

Comment:
**todo:** Please add a comment suggesting we switch to `importlib.metadata` if/when our min version is 3.10.

Reply:
right on!